components/accessRequestForm/requestSection/index.jsx

```diff
@@ -445,7 +445,7 @@ export default function RequestSection(props) {
           <span className={bem("header__content")}>
             {`${agentName} ${getPolicyDetailFromAccess(
               modesObject,
-              "titleConsent"
+              "titleAccess"
```

Comment:
let's name this `titleAccessGrant` (and change also in the function `getPolicyDetailFromAccess` if needed)